client/src/pages/VerifyEmail.js

```diff
@@ -1,11 +1,107 @@
 import React from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+import { mq } from "constants/theme";
 
-const VerifyEmail = () => {
+// ICONS
+import emailVerify from "assets/sign-up-email-verify.svg";
+
+const VerifyEmailContainer = styled.div`
+  display: flex;
+  align-items: center;
+  background-color: #fbfbfd;
+  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    padding-top: 6vh;
+  }
+`;
+
+const VerifyEmailLeftContainer = styled.div`
+  flex-basis: 45%;
+  background-color: #f3f4fe;
+  height: 100vh;
+  position: relative;
+  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    display: none;
+  }
+
+  @media screen and (max-width: ${mq.tablet.narrow.maxWidth}) {
+    flex-basis: 30%;
+  }
+`;
+
+const EnvelopeImageContainer = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 55%;
+  transform: translate(-50%, -50%);
+  width: 80%;
+  margin: 0 auto;
+  @media screen and (max-width: ${mq.tablet.narrow.maxWidth}) {
+    .EnvelopeSVG {
+      width: 100%;
+    }
+  }
+`;
+
+const VerifyEmailRightContainer = styled.div`
+  flex: 1;
+`;
+
+const VerifyDirectionsContainer = styled.div`
+  .directions-header {
+    font-family: Poppins;
+    font-size: 26px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.23;
+    letter-spacing: normal;
+  }
+
+  .directions-text {
+    width: 406px;
+    font-family: Poppins;
+    font-size: 14px;
+    font-weight: 500;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.29;
+    letter-spacing: normal;
+    margin-left: 25%;
+
+    @media screen and (max-width: ${mq.tablet.narrow.maxWidth}) {
+      width: 100%;
+      margin: 0;
+    }
+  }
+`;
+
+const VerifyEmail = ({ email }) => {
   return (
-    <div className="text-center">
-      <h1>Please verify your email</h1>
-    </div>
+    <VerifyEmailContainer>
+      <VerifyEmailLeftContainer>
+        <EnvelopeImageContainer>
+          <img className="EnvelopeSVG" src={emailVerify} alt="" />
```

Comment:
why empty alt tag, maybe just type "verify email" which can help for accessibility

Reply:
True. Missed this. Yeah type something as Robin said. That will also avoid having more console warnings

Reply:
@robinv85 @idjevm added a description to the alt attribute in 27a6b82